ai-ml/emr-spark-rapids/main.tf

```diff
@@ -31,7 +34,7 @@ module "eks" {
   manage_aws_auth_configmap = true
   aws_auth_roles = [
     {
-      rolearn  = module.eks_blueprints_addons.karpenter.iam_role_arn
```

Comment:
`eks_blueprints_addons` keep this name for consistency with EKS Blueprints

Reply:
Checked in all repos and fixed.